source/ChromeDevTools/IChromeSession.cs

```diff
@@ -4,9 +4,13 @@
 
 namespace MasterDevs.ChromeDevTools
 {
+    public interface ICommand<T>
+    {
+
+    }
     public interface IChromeSession
     {
-        Task<ICommandResponse> SendAsync<T>(T parameter, CancellationToken cancellationToken);
+        Task<CommandResponse<TResponse>> SendAsync<TResponse>(ICommand<TResponse> parameter, CancellationToken cancellationToken);
```

Comment:
Is this correct?  There was a reason it was just returning the interface.  I'm having a hard time recalling the reason though.  @qmfrederik do you have thoughts on this?

Reply:
Hmm, now I see there is second implementation of ICommandResponse - ErrorResponse. And it can be returned from the task ..
I'd personally prefer setting task to faulty in such cases than to be forced to check and retype every command. What are your thoughts?

Reply:
I ran through the history via git blame but I don't think I changed this to return an interface, it looks like it just always has been like that. (I did submit a bunch of other PRs to return interfaces in other areas, but not here).

I think this change is OK, since  `CommandResponse<T>` is a very simple class.

Reply:
Ok, i really like this change, i'm just afraid of "breaking" other people using this.